__tests__/propertyVolumeTests.ts

```diff
@@ -3,13 +3,13 @@ process.env['NODE_CONFIG_DIR'] = __dirname + '/config/'
 import * as properties from '../src'
 import * as config from 'config'
 
-properties.addTo(config, '__tests__/testVolumes/testVol/')
-properties.addTo(config, '__tests__/testVolumes/secrets')
+properties.addTo(config, {mountPoint: '__tests__/testVolumes/testVol/'})
```

Comment:
```suggestion
properties.addTo(config, { mountPoint: '__tests__/testVolumes/testVol/' })
```

Reply:
hmm thats not what the ts lint settings are set to in cmc-citizen and postcode I believe